src/Panel/Controller/View/SystemViewController.php

```diff
@@ -0,0 +1,201 @@
+use Kirby\Panel\Ui\View;
+use Kirby\Plugin\Plugin;
+
+/**
+ * Controls the system view
+ *
+ * @package   Kirby Panel
+ * @author    Nico Hoffmann <nico@getkirby.com>
+ * @link      https://getkirby.com
+ * @copyright Bastian Allgeier
+ * @license   https://getkirby.com/license
+ * @since     6.0.0
+ */
+class SystemViewController extends ViewController
+{
+	protected array $exceptions;
+	protected License $license;
+	protected array $plugins;
+	protected System $system;
+	protected UpdateStatus|null $update;
+
+	public function __construct()
+	{
+		parent::__construct();
+
+		$this->system     = $this->kirby->system();
+		$this->license    = $this->system->license();
+		$this->update     = $this->system->updateStatus();
+		$this->exceptions = $this->update?->exceptionMessages() ?? [];
+	}
+
+	public function buttons(): ViewButtons
+	{
+		return ViewButtons::view('system');
+	}
+
+	public function exceptions(): array
+	{
+		if ($this->isDebug() === false) {
+			return [];
+		}
+
+		// Call `::plugins()` to ensure they inject their exceptions
+		$this->plugins();
+
+		return $this->exceptions;
+	}
+
+	protected function isDebug(): bool
+	{
+		return $this->kirby->option('debug', false) === true;
+	}
+
+	protected function isLocal(): bool
+	{
+		return $this->system->isLocal();
+	}
+
+	public function load(): View
+	{
+		return new View(
+			component: 'k-system-view',
+			buttons: $this->buttons(),
+			environment: $this->stats()->reports(),
+			exceptions: $this->exceptions(),
+			info: $this->system->info(),
+			plugins: $this->plugins(),
+			security: $this->security(),
+			urls: $this->urls()
+		);
+	}
+
+	public function plugins(): array
+	{
+		return $this->plugins ??= $this->system->plugins()->values(function (Plugin $plugin) {
+			$authors   = $plugin->authorsNames();
+			$update    = $plugin->updateStatus();
+			$version   = $update?->toArray() ?? $plugin->version() ?? '–';
+
+			// Inject exceptions from plugin to global exceptions
+			$this->exceptions = [
+				...$this->exceptions,
+				...$update?->exceptionMessages() ?? []
+			];
+
+			return [
+				'author'  => $authors === '' ? '–' : $authors,
+				'license' => $plugin->license()->toArray(),
+				'name'    => [
+					'text' => $plugin->name() ?? '–',
+					'href' => $plugin->link(),
+				],
+				'status'  => $plugin->license()->status()->toArray(),
+				'version' => $version,
+			];
+		});
+	}
+
+	public function security(): array
+	{
+		$security = $this->update?->messages() ?? [];
+
+		if ($this->isLocal() === true) {
+			$security[] = [
+				'id'    => 'local',
+				'icon'  => 'info',
+				'theme' => 'info',
+				'text'  => $this->i18n('system.issues.local')
+			];
+		}
+
+		if ($this->isDebug() === true) {
+			$security[] = [
+				'id'    => 'debug',
+				'icon'  => $this->isLocal() ? 'info' : 'alert',
+				'theme' => $this->isLocal() ? 'info' : 'negative',
+				'text'  => $this->i18n('system.issues.debug'),
+				'link'  => 'https://getkirby.com/security/debug'
+			];
```

Comment:
It would be really cool if we could use our Item class here as well. But that's also maybe something for a later PR.